README.md

```diff
@@ -1,3 +1,55 @@
-# go-newrelic-api
+# go-newrelic [![Build Status](https://travis-ci.org/paultyng/go-newrelic.png?branch=master)](https://travis-ci.org/paultyng/go-newrelic)
+
+go-newrelic is a Go SDK for communicating with New Relic APIs.
+
+## Installation
+
+Use `go get` to install go-newrelic.
+
+```bash
+$ go get -u github.com/paultyng/go-newrelic
+```
+
+## Usage
+
+```go
+import "github.com/paultyng/go-newrelic/v4/api"	// with go modules enabled (GO111MODULE=on or outside GOPATH)
+import "github.com/paultyng/go-newrelic/api" // with go modules disabled
+```
+
+Construct a new New Relic client, then use the various methods on the client to access different parts of the New Relic API. For example:
+
+```go
+	config := newrelic.Config{
+		APIKey: "<NEWRELIC_API_KEY>",
+	}
+
+    client := newrelic.New(config)
+
+	config := newrelic.Config{
+		APIKey: "<NEWRELIC_API_KEY>",
+	}
+
+	apps, err := client.ListApplications()
+```
+
+The GoDoc link below contains the full list of available methods.
+
+## Contributing
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are greatly appreciated.
+
+1. Fork the Project
+1. Create your Feature Branch (git checkout -b feature/AmazingFeature)
+1. Commit your Changes (git commit -m 'Add some AmazingFeature')
+1. Push to the Branch (git push origin feature/AmazingFeature)
+1. Open a Pull Request
```

Comment:
Even though markdown orders this correctly, this kind of confuses me as a reviewer. Not necessarily a blocker, but would it better to use the correct numbers in the markdown or format the numbered list in a different way?